### DISCUSSION TITLE
v0.3.8

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#142] [Announcements]
- Fixade SVT Play efter att de ändrat på deras URL.
- Fixade TV4 efter att de också ändrat på grejor.
- Fixade lite på UR också bara för att.

Hoppas det fungerar ett tag nu!

**Full Changelog**: https://github.com/stefansundin/privatkopiera/compare/v0.3.7...v0.3.8

<hr /><em>This discussion was created from the release <a href='https://github.com/stefansundin/privatkopiera/releases/tag/v0.3.8'>v0.3.8</a>.</em>

**Comment by stefansundin** [comment]
Tjena folket.

Förlåt att jag har ignorerat era meddelanden i några dagar. Har verkligen velat ta en titt men har haft så jäkla mycket att göra. Lyckades äntligen hitta lite tid att testa och fixa på Privatkopiera. Det mesta verkar fungera igen men jag har inte testat så himla mycket faktiskt. Skriv en kommentar nedan om det är något som fortfarande är trasigt.

Jag ska iväg i typ en vecka nu så jag kan nog inte svara på några meddelanden på typ två veckor. Hoppas det fungerar tills dess.

Mvh Stefan

**Reply by Shroom86** [reply]
Många tack för detta tillägg och denna uppdatering! :) 

Skulle du ha möjlighet att lägga in stöd för Axess play, när du har tid? 

Tack på förhand

**Reply by stefansundin** [reply]
Nja, det är inget jag använder så jag har inget intresse utav det. Men om någon annan lyckas göra det så tar jag gärna emot den förbättringen.

**Comment by lysol90** [comment]
SVT Play funkar nu igen, men med en liten förändring:

Förut har man kunnat klicka på ett avsnitt men sluppit klicka på "spela" (i det läget där den täcker hela webbläsarfönstret) och ändå kunnat använda privatkopiera. Nu krävs att man klickar på spela-knappen för att tillägget ska förstå att det finns en video att ladda ner. Inga problem, men det var lite frustrerande innan jag fattade varför jag möttes av `Error: NetworkError when attempting to fetch resource.` :) 

Så som sagt, klicka bara på "spela" så att avsnittet hoppar igång så kommer du kunna kopiera kommandot som vanligt.

**Reply by sketch3** [reply]
Mange tak! Afsnit 5 af Husdrömmar Sicilien, som jeg havde hentet på anden vis før Privatkopiera 3.8, viste sig at være h265, som min medieafspiller ikke understøtter. Nu er afsnit 5 [https://www.svtplay.se/video/Kq2bJJ2/husdrommar-sicilien/5-ah-titta-we-have-fire-i-think-we-move-in?id=Kq2bJJ2] hentet med Privatkopiera 3.8. Det er h264, virker fint i medieafspiller. Det var en positiv overraskelse? Desværre går det nok mod HEVC, også på DRTV og NRK TV. Undertekster er genbrugt fra første DL, så jeg ved endnu ikke, hvordan det er i det nye version. Men igen, tak for, at SVT Play virker en tid igen.

**Comment by ETXSLUK** [comment]
Stort tack för v0.3.8 då jag använder Privatkopiera mycket och laddar ner en del från SVT-Play och TV4-Play, dock saknar jag text från TV4-Play. Stort tack igen att det fungerar igen! Men det är lite bökigt.

/Hälsningar Stefan L

**Reply by Lunarwalker60** [reply]
Jag har snickrat ihop ett program som konverterar undertexterna till vanlig .srt. Laddar hem .vtt med svtplay-dl och konverterar sen med mitt program. SVT var inga större problem men TV4 är lurigt iom att de har segmenterade texter vars tidsstämplar refererar till resp audio-segment istället för hela videons startpunkt. svtplay-dl tar hem som en .ts-fil som innehåller tusentals textsegment. Programmet är dock skrivet i mitt "modersmål" VB6 och kräver lite av användaren för att köras i  Windows. Mac har jag ingen möjlighet att utveckla för. Om någon är väldigt sugen på att ha sina TV4-texter omvandlade så kan jag kanske snickra ihop ett separat installationspaket för det om Du själv har svtplay-dl installerat. Edit: Kom att tänka på att Du också behöver ha curl.exe installerat.

**Reply by stefansundin** [reply]
@Lunarwalker60 Ladda upp koden så kanske någon får användning utav den. Det är många som bara lurkar och inte kommenterar eller engagerar sig, men om du publicerar koden så kan folk få användning utav den utan att behöva fråga. :)

Det är typ så jag tänker när jag lägger upp min kod. Även om det inte är någon som har användning av koden just nu så kanske någon hittar den i framtiden.

**Comment by ETXSLUK** [comment]
Hej och tack för svaret!

Jag kör Privatkopiera och ffmpeg men har inte lyckats få svtplay-dl att
fungera med alla hjälpprogram. Numera blir ju .vtt filen tom i SVT-Play men
det finns en workaround som är lite bökig men fungerar. TV4-texter verkar
alltför svåra att få till och de används mer sällan. Ffmpeg fungerar bra
att ladda ner mp4 filer med från SVT-Play men i TV4-play kommer en massa
felmeddelanden blandade under nerladdning och tar lite längre tid, men det
fungerar.

Den tis 29 nov. 2022 kl 15:53 skrev Lunarwalker60 ***@***.***
